RELEASE.md

```diff
@@ -1,3 +1,17 @@
+# Release 0.19.1
+
+## Bug Fixes
+* Fixed wheel issue with tensorflow-io-gcs-filesystem by removing build temporary folder.
```

Comment:
should be "removing temporary build folder"

Reply:
Thanks @kvignesh1420 , the PR has been updated.